From 9bf1f3bdb7c3c40f39e0af0ab3d3024841aff80f Mon Sep 17 00:00:00 2001
From: Anandu Sudhakaran <anansudh@qti.qualcomm.com>
Date: Tue, 29 Mar 2022 13:05:01 +0200
Subject: [PATCH 2/4] pulseaudio.service: Remove dependency to
 pulseaudio.socket.

According to PulseAudio documentation [1] the user-level PulseAudio
service has to be diabled across the whole system. Hence remove
pulseaudio.socket dependency from pulseaudio.service.

[1]: https://wiki.archlinux.org/title/PulseAudio#Starting_system-wide_on_boot

Signed-off-by: Anandu Sudhakaran <anansudh@qti.qualcomm.com>
---
 src/daemon/systemd/user/pulseaudio.service.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/daemon/systemd/user/pulseaudio.service.in b/src/daemon/systemd/user/pulseaudio.service.in
index de19045..c50d98b 100644
--- a/src/daemon/systemd/user/pulseaudio.service.in
+++ b/src/daemon/systemd/user/pulseaudio.service.in
@@ -18,8 +18,6 @@ Description=Sound Service
 Requires=sound.target
 After=sound.target
 
-Requires=pulseaudio.socket
-
 [Service]
 ExecStart=@PA_BINARY@ --daemonize=no \
                       --system \
@@ -31,5 +29,4 @@ Type=notify
 Restart=always
 
 [Install]
-Also=pulseaudio.socket
 WantedBy=multi-user.target
-- 
2.35.1

